lib/api/strapi/operations/getCollection.graphql

```diff
@@ -24,6 +32,14 @@ query getCollectionPageData($whereCollection: JSON, $whereStoreSettings: JSON) {
                parent {
                   title
                   handle
+                  parent {
```

Comment:
Why are we looking farther up the hierarchy now than we were previously?

Reply:
@evannoronha I was extending this to allow for more nesting and see if the query breaks (spoiler alert: if you add even more it does). During today meeting we decided to just show 2 or 3 ancestors in the breadcrumb, so eventually I'll simplify this